RoboJackets/urc-software
# [bug] fixed ros2 controller manager not starting

# Description
This PR does the following:
- Fixes the ros2 control controller manager to allow the rover_drivetrain_controller hardware interface to operate properly

# Testing Steps (if relevant)
1. `colcon build`
2. `. install/setup.bash`
3. `ros2 launch urc_bringup bringup.launch.py`
4. In another terminal type 
`ros2 topic pub /rover_drivetrain_controller/cmd_vel geometry_msgs/msg/TwistStamped "{twist: {linear: {x: 1}}}" -r 100`
to send a drivetrain command, and you should see a response in the first terminal

Expectation: Should start all hardware interfaces and allow you to drive the drivetrain

# Self Checklist
